frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -672,6 +674,17 @@ export const dashboardLogic = kea<dashboardLogicType>([
                 return candidateShortest?.isValid() ? candidateShortest : null
             },
         ],
+        blockRefresh: [
+            (s) => [s.lastRefreshed],
```

Comment:
should we persist this value so a refresh of the page doesn't let you refresh (although the canny user could delete the value from storage, but if they're that determined 🤷 )

Reply:
this is based on `lastRefreshed` which is derived from the `last_refresh` value from insights that comes the backend so should hold across page refreshes, right?

Reply:
Yep, I read just the diff and made bad assumptions 🫠